tests/tasks/dremio/test_dremio.py

```diff
@@ -1,20 +1,47 @@
 import pytest
+import os
 from pyarrow.flight import FlightUnavailableError
 
 from prefect.tasks.dremio.dremio import DremioFetch
 
 
 class TestDremioFetch:
     def test_construction(self) -> None:
-        task = DremioFetch(user="test", password="test", host="test")
-        assert (task.user == "test") and (task.host == "test")
+        task = DremioFetch(username="test", password="test", hostname="test")
```

Comment:
I prefer to pass different values for each of these so your test ensures they are set correctly

Reply:
Good call. Done.